frontend/src/scenes/project/Settings/DangerZone.tsx

```diff
@@ -28,19 +28,22 @@ export function DeleteProject(): JSX.Element {
         })
     }
 
-    const DeleteButton = (
-        <Button type="primary" danger onClick={handleClick}>
-            Delete Project
-        </Button>
-    )
-
-    let deletionDisabledReason: string | null = null
+    let accessRestrictionReason: string | null = null
     if ((currentOrganization?.membership_level ?? -1) < OrganizationMembershipLevel.Admin)
-        deletionDisabledReason = 'You must be an organization administrator to delete projects.'
+        accessRestrictionReason = 'This section is restricted to administrators.'
 
-    return (
-        <div>
-            {deletionDisabledReason ? <Tooltip title={deletionDisabledReason}>{DeleteButton}</Tooltip> : DeleteButton}
+    return accessRestrictionReason ? (
+        <div className="access-restricted">
+            <LockOutlined className="text-warning" />
+            {accessRestrictionReason}
+        </div>
+    ) : (
+        <div className="mt">
+            <Button type="primary" danger onClick={confirmDeleteProject} className="mr-05">
+                Delete Project
+            </Button>
+            This will permanently delete your project plus <b>all the data and events</b> associated to it. Please be
```

Comment:
It won't actually _delete_ the events, since we have a hard time removing data from ClickHouse and that's not implemented at the moment.

Reply:
Oh right, opening a separate issue to track this and updating copy.